insights_dart/lib/algolia_insights.dart

```diff
@@ -1,4 +1,7 @@
 /// Algolia Insights Helper
 library algolia_helper_flutter;
 
-export 'src/hits_tracker.dart';
+export 'src/event_tracker.dart';
+export 'src/filter_event_tracker.dart';
+export 'src/filter_event_tracker_adapter.dart';
+export 'src/insights.dart';
```

Comment:
Since this is an new package, the CI should be updated

Reply:
Will handle it before the release.